internal/install/application_configuration_test.go

```diff
@@ -60,56 +60,56 @@ func TestSelectElasticAgentImageName_DefaultImage816_WithoutEnvVar(t *testing.T)
 	os.Unsetenv(disableElasticAgentWolfiEnvVar)
 
 	selected := selectElasticAgentImageName(version, "")
-	assert.Equal(t, selected, elasticAgentWolfiImageName)
+	assert.Equal(t, elasticAgentWolfiImageName, selected)
 }
 
 func TestSelectElasticAgentImageName_DisableWolfiImageEnvVar(t *testing.T) {
 	version := stackVersion8160
 	t.Setenv(disableElasticAgentWolfiEnvVar, "true")
 	selected := selectElasticAgentImageName(version, "")
-	assert.Equal(t, selected, elasticAgentCompleteImageName)
+	assert.Equal(t, elasticAgentCompleteImageName, selected)
 }
 func TestSelectElasticAgentImageName_EnableWolfiImageEnvVar(t *testing.T) {
 	version := stackVersion8160
 	t.Setenv(disableElasticAgentWolfiEnvVar, "false")
 	selected := selectElasticAgentImageName(version, "")
-	assert.Equal(t, selected, elasticAgentWolfiImageName)
+	assert.Equal(t, elasticAgentWolfiImageName, selected)
 }
 
 func TestSelectCompleteElasticAgentImageName_ForceCompleteImage(t *testing.T) {
 	version := stackVersion8160
 	selected := selectElasticAgentImageName(version, "complete")
-	assert.Equal(t, selected, elasticAgentCompleteImageName)
+	assert.Equal(t, elasticAgentCompleteImageName, selected)
 }
 
 func TestSelectCompleteElasticAgentImageName_ForceDefaultImage_DisabledEnvVar(t *testing.T) {
 	version := stackVersion8160
 	t.Setenv(disableElasticAgentWolfiEnvVar, "true")
 	selected := selectElasticAgentImageName(version, "default")
-	assert.Equal(t, selected, elasticAgentCompleteImageName)
+	assert.Equal(t, elasticAgentCompleteImageName, selected)
 }
 
 func TestSelectCompleteElasticAgentImageName_ForceDefaultImage_EnabledEnvVar(t *testing.T) {
 	version := stackVersion8160
 	t.Setenv(disableElasticAgentWolfiEnvVar, "false")
 	selected := selectElasticAgentImageName(version, "default")
-	assert.Equal(t, selected, elasticAgentWolfiImageName)
+	assert.Equal(t, elasticAgentWolfiImageName, selected)
 }
 
 func TestSelectCompleteElasticAgentImageName_ForceDefaultImageOldStack(t *testing.T) {
 	version := "8.15.0-SNAPSHOT"
 	selected := selectElasticAgentImageName(version, "default")
-	assert.Equal(t, selected, elasticAgentCompleteImageName)
+	assert.Equal(t, elasticAgentCompleteImageName, selected)
 }
 
 func TestSelectCompleteElasticAgentImageName_ForceSystemDImage(t *testing.T) {
 	version := stackVersion8160
 	selected := selectElasticAgentImageName(version, "systemd")
-	assert.Equal(t, selected, elasticAgentImageName)
+	assert.Equal(t, elasticAgentImageName, selected)
```

Comment:
Updated the order to ensure that the values are the expected and the actual ones as required by the assert functions.
https://pkg.go.dev/github.com/stretchr/testify/assert#Equal